app/server.go

```diff
@@ -1745,40 +1753,41 @@ func (s *Server) doLicenseExpirationCheck() {
 		if appErr != nil {
 			mlog.Debug(appErr.Error())
 		}
-	}
 
-	if !license.IsPastGracePeriod() {
-		mlog.Debug("License is not past the grace period.")
 		return
 	}
 
-	//send email to admin(s)
-	for _, user := range users {
-		user := user
-		if user.Email == "" {
-			mlog.Error("Invalid system admin email.", mlog.String("user_email", user.Email))
-			continue
+	if license.IsPastGracePeriod() {
```

Comment:
Hmm .. any reason to invert the check? This creates more indented code. In Go, we prefer early returns.